client/reader/share/index.jsx

```diff
@@ -43,27 +46,28 @@ var actionMap = {
 		window.open( fackbookUrl, 'facebook', 'width=626,height=436,resizeable,scrollbars' );
 	},
 	pressThis: function( post ) {
-		var primarySite = sitesList.getPrimary(), wordpressUrl;
-
+		const primarySite = sitesList.getPrimary();
 		if ( ! primarySite ) {
 			return;
 		}
 
-		if ( primarySite.wpcom_url && ! primarySite.jetpack ) {
-			wordpressUrl = url.parse( 'https://' + primarySite.wpcom_url + '/wp-admin/press-this.php' );
-		} else {
-			wordpressUrl = url.parse( url.resolve( primarySite.URL, 'wp-admin/press-this.php' ) );
+		const args = {};
+
+		if ( post.content_embeds && post.content_embeds.length ) {
+			args.embed = post.content_embeds[0].src;
```

Comment:
`post.content_embeds` has already been vetted and validated, back when it was constructed. The validation on the recieving end in the editor is really more because any old third party could make up a URL that prefills the editor with junk and I wanted to limit the junk to urls.